README.md

```diff
@@ -95,7 +95,7 @@ To get involved in Apache Beam:
 * [Report an issue](https://github.com/apache/beam/issues/new/choose).
 
 Instructions for building and testing Beam itself
-are in the [contribution guide](https://beam.apache.org/contribute/).
+are in the [contribution guide](./CONTRIBUTING.md).
```

Comment:
nit: (though idk if this will make a difference): include full URL to the contributing.md file?

Reply:
The (minor) advantage of doing it this way is that if someone is looking at the repo locally in their editor, this will allow them to continue to view it locally without being redirected to a browser. It also means that they will stay on the same branch when they view it (for better or for worse)